src/app/AlwaysOn.UI/package.json

```diff
@@ -19,7 +19,8 @@
     "@vue/cli-service": "^5.0.4",
     "is-primitive": "^3.0.1",
     "nprogress": "^0.2.0",
-    "vue-template-compiler": "^2.6.14"
+    "vue-template-compiler": "^2.6.14",
```

Comment:
why is this in this PR?

Reply:
I presume adopted from #335 to make it work for Heyko.

Reply:
This comes from PR #335 to make the PR run again. Merged PR #335 into this PR.

Reply:
ok then lets get #335 merged into main and rebase this one :)